server/models/ticketSchema.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const ticketSchema = new mongoose.Schema(
+  {
+    price: {
+      type: Number,
+      required: [true, "Ticket price is required"],
+    },
+    ticketClass: {
```

Comment:
I dont know what is better - use class in seat or in ticket. Maybe class in seat is better.